OPSEXP-3310 Add known issue for aps artifact fetch (#211)
Alfresco/alfresco-dockerfiles-bakery: README.md

For authentication check out [nexus authentication](#nexus-authentication)

## Known issues

> - While building `all` target with `make`, which is meant to build the `acs` the
>   fetching script will also try to download the artifacts meant for the `aps`.
>   Build will proceed as it will use the `default` bake target which points to
>   `enterprise` and `community` targets. To workaround this issue you can change
>   the filenames of artifacts files in the `aps` folder to anything else than
>   `artifacts-*.yaml`.

Reviewer: ```suggestion
- When building `all` target with `make`, which is meant to build only `acs`, the
  fetching script will also try to download the artifacts related to `aps`.
  Build will proceed as it will use the `default` bake target which points to
  `enterprise` and `community` targets. To workaround this issue you can change
  the filenames of artifacts files in the `aps` folder to anything else than
  `artifacts-*.yaml`.
```

>   `artifacts-*.yaml`.

Reviewer: not sure if the workaround is really worth mentioning, we are documenting this but there is really no side effect apart from spending time in downloading an artifact which is not needed (which is then stored in the cache so downloaded only once)
